<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>615</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>625</width>
    <height>615</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>625</width>
    <height>615</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>May the rngesus be on your side</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="SKILLS">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>461</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="Group_INTELLIGENCE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>321</height>
      </rect>
     </property>
     <property name="title">
      <string>INTELLIGENCE</string>
     </property>
     <widget class="QTableWidget" name="Intelligence_Table">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>141</width>
        <height>301</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>11</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>23</number>
      </attribute>
      <row>
       <property name="text">
        <string>Awareness</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Business</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Deduction</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Education</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Elder Speech</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Nilgaardian</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Skellige</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Nordling</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Monster Lore</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Streetwise</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Tactics</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Teaching</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Wilderness Survival</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>x</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>1</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="0">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>1</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>4</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>2</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string>2</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="11" column="0">
       <property name="text">
        <string>1</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Group_DEXTERITY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>DEXTERITY</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Group_CRAFTING">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>141</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>CRAFTING</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Group_BODY">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>BODY</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Group_REFLEX">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>250</y>
       <width>141</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>REFLEX</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Group_EMPATHY">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>141</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>EMPATHY</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Group_WILL">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>141</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>WILL</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="Commandline">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>411</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>This is where the copyable command appears!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CopyToClipBoard">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>520</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>181</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Copy to Clipboard</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="STATS_TABS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Stats_tab1">
     <attribute name="title">
      <string>Stats</string>
     </attribute>
     <widget class="QGroupBox" name="STATS_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="currentStats" native="true">
       <property name="geometry">
        <rect>
         <x>99</x>
         <y>0</y>
         <width>21</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTabWidget" name="VARIABLE_TABS">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="VARIABLES_tab1">
       <attribute name="title">
        <string>Variables</string>
       </attribute>
       <widget class="QLabel" name="label_health">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>47</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Health</string>
        </property>
        <property name="buddy">
         <cstring>value_health</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="label_drunkenness">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Drunkenness</string>
        </property>
        <property name="buddy">
         <cstring>value_drunkenness</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="label_luck">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>47</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Luck</string>
        </property>
        <property name="buddy">
         <cstring>value_luck</cstring>
        </property>
       </widget>
       <widget class="QSpinBox" name="value_health">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>/5</string>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="value_drunkenness">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="suffix">
         <string>/5</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="value_luck">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="VARIABLES_tab2">
       <attribute name="title">
        <string>Souls</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>141</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Control Panel</string>
      </property>
      <widget class="QCheckBox" name="WellRested">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Well Rested</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="WeddFed">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Well Fed</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="Other_Buffs">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-99</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_otherBuffs">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Other Buffs</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ResetLuck">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset Luck</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Reputation">
     <attribute name="title">
      <string>Reputation</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="group_basic" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QPushButton" name="dice_d10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1d10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dice_d20">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1d20</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dice_d100">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1d100</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dice_d50">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1d50</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStats">
    <property name="title">
     <string>Stats</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionGrimoire"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuStats"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save values to a file</string>
   </property>
  </action>
  <action name="actionGrimoire">
   <property name="text">
    <string>Grimoire</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
